### PR TITLE
[AR] Improve the Arabic language responses

### PR DESCRIPTION
**Pull request by omaramin-2000**
Improve the Arabic language responses

### CODE REVIEW

File: sentences/ar/_common.yaml [hunk 1]
```diff
@@ -115,9 +115,9 @@ expansion_rules:
   switch: "[ال](مفتاح|مفاتيح)"
   fan: "[ال](مروحة|مراوح|مروحات)"
 
-  turn: "(غير|اضبط|ضبط)"
-  turn_on: "(افتح|شغل|اشعل|اشغل|ابدأ|شعل|فعل|تفعيل|تشغيل|فعل)"
-  turn_off: "(اغلق|اطفئ|طفي)"
+  turn: "(غير|اضبط||أضبط||إضبط|ضبط)"
```
Comment (Natfaji) [inline sentences/ar/_common.yaml:118]: double pipes "||" are unnecessary.

File: sentences/ar/_common.yaml [hunk 2]
```diff
@@ -115,9 +115,9 @@ expansion_rules:
...
+  turn_on: "(افتح|شغل|اشعل|أشعل|اشغل||أشغل|ابدأ||إبدأ|شعل|فعل|تفعيل|تشغيل|فعل)"
```
Comment (Natfaji) [inline sentences/ar/_common.yaml:119]: double pipes "||" are unnecessary.

File: sentences/ar/_common.yaml [hunk 3]
```diff
@@ -115,9 +115,9 @@ expansion_rules:
...
+  turn_off: "(أغلق|اغلق|اطفئ||أطفئ|طفي)"
```
Comment (Natfaji) [inline sentences/ar/_common.yaml:120]: double pipes "||" are unnecessary.

**Review by Natfaji — CHANGES_REQUESTED** [review]

**Comment by omaramin-2000** [comment]
Alright, I need help with the modifications cause the test fails, can you tell me what else I need to modify?

File: tests/ar/light_HassLightSet.yaml [hunk 4]
```diff
@@ -120,7 +168,11 @@ tests:
   # color individual light
   - sentences:
       - "اضبط لون ضوء المطبخ الرئيسي الى احمر"
```
Comment (Natfaji) [inline tests/ar/light_HassLightSet.yaml:170]: the problem is that the word "احمر" is misspelled, it should be "أحمر". this also applies for all other "احمر" is this file

File: tests/ar/light_HassLightSet.yaml [hunk 5]
```diff
@@ -147,7 +207,11 @@ tests:
   # color all lights
   - sentences:
       - "اضبط لون جميع الاضواء الى احمر"
```
Comment (Natfaji) [inline tests/ar/light_HassLightSet.yaml:209]: same

File: tests/ar/light_HassLightSet.yaml [hunk 6]
```diff
@@ -132,9 +184,17 @@ tests:
   # color specific area
   - sentences:
       - "اضبط لون جميع الاضواء الى احمر في المطبخ"
```
Comment (Natfaji) [inline tests/ar/light_HassLightSet.yaml:186]: same

**Review by Natfaji — CHANGES_REQUESTED** [review]

**Review by Natfaji — CHANGES_REQUESTED** [review]
add both way to say red ("أحمر" and "احمر") to each response

**Review by Natfaji — APPROVED** [review]

**Review by Ahmed-farag36 — APPROVED** [review]

File: sentences/ar/_common.yaml [hunk 7]
```diff
@@ -115,9 +115,9 @@ expansion_rules:
   switch: "[ال](مفتاح|مفاتيح)"
   fan: "[ال](مروحة|مراوح|مروحات)"
 
-  turn: "(غير|اضبط|ضبط)"
-  turn_on: "(افتح|شغل|اشعل|اشغل|ابدأ|شعل|فعل|تفعيل|تشغيل|فعل)"
-  turn_off: "(اغلق|اطفئ|طفي)"
+  turn: "(غير|اضبط||أضبط|إضبط|ضبط)"
```
[the remark this answers is not in the document]
Comment (Natfaji) [reply sentences/ar/_common.yaml:118]: missed that one

**Review by Ahmed-farag36 — APPROVED** [review]

**Pull request merged**